### ISSUE TITLE
Links in XML response incorrectly detected

### ISSUE THREAD

**Issue by kingstevo** [a85/POSTMan-Chrome-Extension#2481]
1. Postman Version: 4.8.3
 2. App (Chrome app or Mac app): Chrome app - Chrome 54.0.2840.99
 3. OS details: win / x86-64

- Make an API call as usual
- API responds with XML
- Links are in CDATA enclosed fields
- Postman correctly identifies the links and highlights them in the 'Pretty' view
- Clicking on the link opens a new Postman tab
- The two ]] that mark the closing of the CDATA field are incorrectly appended to the link

e.g. example XML response in the 'Pretty' tab:
```
 <Impression>
<![CDATA[https://ads-prod02.amillionads.com/tracking/20a52976c809c45fcaa462cff03b1a7a/mqpH1/Ad5832c2d134abb0.95619726/impression]]>
</Impression>
```

Clicking the link should go to:
`https://ads-prod02.amillionads.com/tracking/20a52976c809c45fcaa462cff03b1a7a/mqpH1/Ad5832c2d134abb0.95619726/impression`
But Postman incorrectly goes to:
`https://ads-prod02.amillionads.com/tracking/20a52976c809c45fcaa462cff03b1a7a/mqpH1/Ad5832c2d134abb0.95619726/impression]]`

**Comment by vegetableman** [comment]
@kingstevo Thanks for posting this issue 👍 . Will get back to you.

**Comment by vegetableman** [comment]
@kingstevo The issue has been fixed and is part of the latest release `4.10.5`. Feel free to reopen if it still exists. ^_^
